src/components/shared/VirtualList.js

```diff
@@ -100,7 +98,7 @@ class VirtualListInner extends React.PureComponent<VirtualListInnerProps> {
     if (this._container) {
       return this._container.getBoundingClientRect();
     }
-    return new window.DOMRect();
+    return new DOMRect(0, 0, 0, 0);
```

Comment:
I believe you'll want to remove `DOMRect` from `src/types/globals/Window.js` as well.